src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -324,7 +324,7 @@ protected static boolean referenceAlleleMatchesReferenceForIndel(final List<Alle
      *    Note: this will modify the start/stop and alleles of this builder.
      *    Also note: if the reference allele does not match the reference sequence, this method will throw an exception
      *
-     *    Based on Adrian Tan, Gonçalo R. Abecasis and Hyun Min Kang. (2015)
+     *    Based on Adrian Tan, Goncalo R. Abecasis and Hyun Min Kang. (2015)
```

Comment:
I would recommend to use `&ccedil;` instead, to render as HTML and maintain the original name.